### ISSUE TITLE
[Bug]:  The execution result is not as expected.

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#6715]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

commit id: bb9c3edb9ba7f69bcaa1c28d553ed6eeef87aecd

use mo_catalog;

select * from mo_columns where att_database = 'mo_catalog';   95 rows in set (0.03 sec)

select count(\*) from mo_columns where att_database = 'mo_catalog';
+----------+
| count(*) |
+----------+
|        0 |
+----------+

**Comment by nnsgmsone** [comment]
The mo_catalog field given by dn is not set. So it can't be query.

**Comment by ouyuanning** [comment]
PR is waiting for review

**Comment by nnsgmsone** [comment]
fixed by #6742

**Comment by tianyahui-python** [comment]
<img width="1349" alt="截屏2022-11-21 10 21 02" src="https://user-images.githubusercontent.com/39303074/202947171-fcb49628-198c-423c-9874-2479d0418691.png">
